pkgs/development/python-modules/jaxtyping/default.nix

```diff
@@ -16,14 +16,14 @@
 let
   self = buildPythonPackage rec {
     pname = "jaxtyping";
-    version = "0.2.21";
-    format = "pyproject";
+    version = "0.2.22";
+    pyproject = true;
```

Comment:
did the recommended way of doing this change? i was under the impression that `format = "pyproject";` was still the way to go?

Reply:
From what I understood yes: https://github.com/NixOS/nixpkgs/issues/253154